Backend/src/main/java/com/br/sga/service/AlunoService.java

```diff
@@ -0,0 +1,57 @@
+package com.br.sga.service;
+
+import com.br.sga.ApplicationException.EmptyListException;
+import com.br.sga.ApplicationException.RecordNotFoundException;
+import com.br.sga.domain.Aluno;
+import com.br.sga.repository.AlunoRepository;
+import com.br.sga.service.dto.AlunoDTO;
+import com.br.sga.service.mapper.AlunoMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+@Transactional
+public class AlunoService {
+    private final AlunoRepository repository;
+    private final AlunoMapper mapper;
+
+    public List<AlunoDTO> buscarTodos() {
```

Comment:
Não é necessário verificar se a lista e vazia e tambem nao precisa lançar uma exeption para isso, pois se for um novo primeiro cadastro de tudo ele vai ficar lançando exepiton desnecessárias.. por enquanto pode seguir esse padrao 

    public List<AlunoDTO> buscarTodos() {
        return mapper.toDto(repository.findAll());
    }
    
    nao precisa verificar se é vazia